WordPress/src/androidTest/java/org/wordpress/android/e2etests/ComposeTests.java

```diff
@@ -0,0 +1,105 @@
+package org.wordpress.android.e2etests;
+
+
+import android.support.test.espresso.contrib.ViewPagerActions;
+import android.support.test.rule.ActivityTestRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.wordpress.android.R;
+import org.wordpress.android.e2etests.utils.TestData;
+import org.wordpress.android.e2etests.utils.ToastMatcher;
+import org.wordpress.android.ui.WPLaunchActivity;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static android.support.test.espresso.action.ViewActions.replaceText;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+public class ComposeTests {
+    @Rule
+    public ActivityTestRule<WPLaunchActivity> mActivityTestRule = new ActivityTestRule<>(WPLaunchActivity.class);
+
+    private TestData mData = new TestData();
+    private String mUserEmail = mData.getEmail();
```

Comment:
Rather minor: Do we plan to modify `mUserEmail`'s value perhaps? If not, can we use `mData.getEmail()` directly and remove the variable altogether?